client/my-sites/upgrades/components/domain-warnings/pending-gapps-tos-notice.jsx

```diff
@@ -0,0 +1,118 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal Dependencies
+ */
+import analyticsMixin from 'lib/mixins/analytics';
+import Notice from 'components/notice';
+import NoticeAction from 'components/notice/notice-action';
+import support from 'lib/url/support';
+
+const learnMoreLink = <a href={ support.COMPLETING_GOOGLE_APPS_SIGNUP } target="_blank" />,
+	strong = <strong />;
+
+const PendingGappsTosNotice = React.createClass( {
+	mixins: [ analyticsMixin( 'domainManagement', 'googleApps' ) ],
+
+	propTypes: {
+		domains: React.PropTypes.array.isRequired,
+		section: React.PropTypes.string.isRequired
+	},
+
+	getGappsLoginUrl( email, domain ) {
+		return `https://accounts.google.com/AccountChooser?Email=${ email }&service=CPanel&continue=https%3A%2F%2Fadmin.google.com%2F${ domain }%2FAcceptTermsOfService%3Fcontinue%3Dhttps%3A%2F%2Fmail.google.com%2Fmail%2Fu%2F1`;
+	},
+
+	getNoticeSeverity() {
+		const subscribedDaysAgo = days => {
+			return domain => this.moment( domain.googleAppsSubscription.subscribedDate ).isBefore( this.moment().subtract( days, 'days' ) );
+		};
+
+		if ( this.props.domains.some( subscribedDaysAgo( 21 ) ) ) {
+			return 'error';
+		} else if ( this.props.domains.some( subscribedDaysAgo( 7 ) ) ) {
+			return 'warning';
+		}
+
+		return 'info';
+	},
+
+	generateLogInClickHandler( { domain, user, severity, isMultipleDomains } ) {
+		return () => {
+			this.recordEvent( 'pendingAccountLogInClick', { domain, user, severity, isMultipleDomains, section: this.props.section } );
+		};
+	},
+
+	oneDomainNotice() {
+		const severity = this.getNoticeSeverity(),
+			domain = this.props.domains[0].name,
+			users = this.props.domains[0].googleAppsSubscription.pendingUsers;
+
+		return (
+			<Notice
+				status={ `is-${ severity }` }
+				showDismiss={ false }
+				key="pending-gapps-tos-acceptance-domain"
+				text={ this.translate(
+					'You\'re almost there! To activate your email {{strong}}%(emails)s{{/strong}}, please log in to Google Apps and finish setting it up. {{learnMoreLink}}Learn More.{{/learnMoreLink}}',
```

Comment:
This PR lingered for long so I don't want to create a blocker, but I think it'd be great to have a different copy for different severities: 

![image](https://cloud.githubusercontent.com/assets/991022/15762585/7473df36-28d3-11e6-9a76-637fb4ee3423.png)

This kinda feels awkward. It's red but it's suggestive.